app/views/spam2/_spam.html.erb

```diff
@@ -36,31 +38,37 @@
         <a class="dropdown-item" href="/dashboard"><i class="fa fa-list-alt text-primary"></i> Dashboard</a>
       <div class="dropdown-divider"></div>  
         <h6 class="dropdown-header">Spam Moderation</h6>
-        <a class="dropdown-item <% unless  params[:action] =="_spam_comments" || params[:action] =="_spam_revisions" || params[:action] =="_spam_flags" || params[:type] == "wiki" || params[:action] == "_spam_users"%> active <% end %>" href="/spam2/filter/all/30"><i class="drop fa fa-address-book text-primary"></i> Bulk Moderation</a>
-        <a class="dropdown-item <% if params[:type] == "wiki" %> active <% end %>" href="/spam2/filter/wiki/50" ><i class="drop fa fa-book text-primary"></i> Wiki</a>
+        <a class="dropdown-item <% unless  params[:action] =="_spam_comments" ||params[:type] =="queue" || params[:action] =="_spam_revisions" || params[:action] =="_spam_flags" || params[:type] == "wiki" || params[:action] == "_spam_users"%> active <% end %>" href="/spam2/filter/all/30"><i class="drop fa fa-address-book text-primary"></i> Bulk Moderation</a>
+        <a class="dropdown-item <% if params[:type] == "wiki" %> active <% end %>" id="wiki_drop" href="/spam2/filter/wiki/30" ><i class="drop fa fa-book text-primary"></i> Wiki</a>
+        <a class="dropdown-item <% if params[:type] == "queue" %> active <% end %>" id="queue_drop" href="/spam2/filter/queue/30" ><i class="drop fa fa-clock text-primary"></i> Queue</a>
         <a class="dropdown-item <% if params[:action] == "_spam_revisions" %> active <% end %>" href="/spam2/revisions"><i class="drop fa fa-list text-primary"></i> Revision</a>
         <a class="dropdown-item <% if params[:action] == "_spam_comments" %> active <% end %>" href="/spam2/comments/filter/all/30" ><i class="drop fa fa-comment text-primary"></i> Comments</a>
         <a class="dropdown-item <% if params[:action] == "_spam_flags" %> active <% end %>"  href="/spam2/flags/filter/all/30" ><i class="drop fa fa-flag text-primary"></i> Flag </a>
         <a class="dropdown-item  <% if params[:action] == "_spam_users" %> active <% end %>" href="/spam2/users/filter/all/30"><i class="drop fa fa-user text-primary"></i> User moderation</a>
       </div>
     </div>
 </div>
-
+<script>
```

Comment:
Could you please indent properly this whole script? Thanks,

Reply:
Done, Thanks!!